src/config.js

```diff
@@ -1 +1,14 @@
-export { default } from "./configs/config.aws.js";
+// The build script injects this global variable if "local.config.js"
+// exists in the project root (alongside package.json). That file is in
+// gitignore and so can be modified without making changes to this file.
+
+const importConfig = () => {
+  if (typeof CONFIG_PATH !== "undefined") {
+    return import(CONFIG_PATH);
+  } else {
+    return import("./configs/config.aws.js");
```

Comment:
Important thing here is that syntactically these are still imports of string literals (since CONFIG_PATH gets replaced by a string literal before being built), so they get bundled normally and do not emit import() calls.